web/libs/datamanager/src/components/DataManager/Toolbar/LabelButton.jsx

```diff
@@ -84,30 +86,30 @@ export const LabelButton = injector(({ store, canLabel, size, target, selectedCo
   return canLabel ? (
     <Interface name="labelButton">
       <div>
-        <div style={{ display: "flex" }}>
+        <ButtonGroup>
           <Button
-            size={size}
```

Comment:
Why did you remove `size` from here? I'm not sure it's actually used, but all `instruments` in DM's Toolbar use `size` at least in the code.